packages/manager/apps/pci-ai-endpoints/src/components/utils/getUserRole.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect, useMemo, useContext } from 'react';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+
+interface UseUserInfos {
+  getUserRole(): string;
+}
+
+const useUserInfos = (): UseUserInfos => {
+  const { shell } = useContext(ShellContext);
+  const [role, setRole] = useState<string>('');
+
+  useEffect(() => {
+    shell.environment.getEnvironment().then((env) => {
+      const roles = env.user?.auth?.roles;
+      setRole(roles && roles.length > 0 ? roles[0] : '');
```

Comment:
```suggestion
      setRole(env.user.auth.roles[0] || '');
```
Typing says it cannot be null